pom.xml

```diff
@@ -57,6 +57,8 @@
     <project.junit.version>4.12</project.junit.version>
     <project.guava.version>27.1-android</project.guava.version>
```

Comment:
this should go to 28.0-android in a separate PR

Reply:
In #300